### DISCUSSION TITLE
Can you add a hyperlink in github issue title?

### DISCUSSION THREAD

**Discussion by mirnawong1** [community/community#42466] [New to GitHub]
Hi there, new-ish to github and wanted to double check if it's possible to hyperlink a URL in a github issue or pr? I know you can add the full link but it looks pretty disorganized so i wanted to see if it was possible to link it like the example below:

- This is my GitHub [issue title](https://docs.getdbt.com/)

Many thanks!

**Comment by jfullstackdev** [comment]
I don't know if there are external services for that, but here is the official documentation 

https://docs.github.com/en/get-started/writing-on-github/working-with-advanced-formatting/autolinked-references-and-urls

it's only Markdown in the body but not the title itself

**Comment by Lenochxd** [comment]
For the next people who see this post, no you can't. I just tested.
